website/common/script/libs/shops.js

```diff
@@ -428,6 +430,8 @@ shops.getSeasonalShop = function getSeasonalShop (user, language) {
       text: i18n.t(seasonalShopConfig.featuredSet),
       items: shops.getSeasonalGearBySet(user, seasonalShopConfig.featuredSet, officialPinnedItems, language, true),
     },
+    // return / subscribe to user._V, to let vue refresh the seasonal shop overview
+    _v: myUserVersion,
```

Comment:
I think this is returned by default under `userV`as a top level response property (next to `success` and `data`)

Reply:
I just used this to store the `myUserVersion` so that I don't get an `unused-var` lint error, I could remove this and just have the following:


```
  // eslint-disable-next-line
  const myUserVersion = user._v;
```

this seems to work with it too

Reply:
Ok I think you can remove the changes in this file and it'll work anyway because user._v is returned anyway in the response top level object

Reply:
yeah I don't need the return value at all, I just need to "subscribe" in the vue-way (see line [420](https://github.com/HabitRPG/habitica/pull/10971/files/b3d3d5d5cffef06696569858cebf9c47025f19f6#diff-bbbfee4959388a7be92457cbcc12434fR420) ) so that I can get the updates inside the web app, without it it'll stay "unchanged" for vue

Reply:
could this be moved to a clien side file only with a comment on why it's needed? maybe where it calls the common function? Otherwise I'd still add a more exhaustive comment on why it's needed and I'd remove it from the return value as it's confusing

Reply:
subscribed in the vue-file itself, somehow I thought it would be better to do inside the lib^^, should be good now